docs/books/modules/user-guide/creating-vhost-policies.adoc

```diff
@@ -54,6 +55,11 @@ The literal hostname of the vhost (the messaging endpoint) or a pattern that mat
 +
 If `enableVhostNamePatterns` is set to `true`, you can use wildcards to specify a pattern that matches a range of hostnames. For more information, see xref:vhost-policy-hostname-pattern-matching-rules-{context}[].
 
+`aliases`::
+Alternative literal hostnames or patterns that direct the router to use the settings in this vhost. Alias hostnames that match an incoming connection will use the settings defined in the contained vhost. In a multi-tenant configuration a connection to a vhost alias will use the base vhost hostname for the tenant namespace. In this example if a connection is directed to vhost `example.org` then the settings from the base vhost hostname `example.com` will apply and `example.com` will be the tenant namespace. Vhost `hostname` and `aliases` settings from all vhosts are shared in a single list and must be unique.
```

Comment:
Alternative literal hostnames or patterns that direct the router to use the settings in this vhost. 
Alias hostnames that match an incoming connection use the settings defined in the vhost section. 
In a multi-tenant configuration, a connection to a vhost alias uses the base vhost hostname for the tenant namespace. 
In this example if a connection is directed to vhost `example.org` then the settings from the base vhost hostname `example.com` apply and `example.com` becomes the tenant namespace. 
Vhost `hostname` and `aliases` settings from all vhosts must be unique.

@ChugR  Not sure if my second sentence is correct?

Reply:
Yes, I think it's fine. Please submit that as a separate PR. Thanks!